packages/x-tree-view/src/internals/index.ts

```diff
@@ -0,0 +1,17 @@
+export { useTreeView } from './useTreeView';
```

Comment:
We will move these elements outside of the `internals` folder, but having them centralized in this files help me know which internals element will need to be public before `useTreeView` becomes public AND avoid very deep imports in the the demos of the new page (which causes lint errors).